tools/scripts/vsc_intellisense.mk

```diff
@@ -25,15 +26,19 @@ INCLUDE_CORE_PATTERN 	= $(PROJECT_BUILD)/$(PROJECT_NAME)
 INCLUDE_CORE_CORRECTED	= $(PROJECT)
 INCLUDE_NOOS_PATTERN	= $(PROJECT_BUILD)/noos
 INCLUDE_NOOS_CORRECTED	= $(NO-OS)
+INCLUDE_MAXIM_PATTERN	= $(PROJECT_BUILD)/root/MaximSDK/Libraries
```

Comment:
ideally, this file should not contain and Maxim or STM32 related stuff, so maybe think of a way to move the specific stuff to maxim.mk, stm32.mk

Reply:
i think there were some stm32 related things in here because it wasn't critical at that point, but now that we have 2 platforms that use this file, we can easily separate

Reply:
@EoghanMonahan , any comment regarding @buha remark?

Reply:
I have since updated this to make it more generalized and not as Maxim specific, so the same setup could be used for any projects with other library locations

Reply:
I see you have updated it in the last commit... in general, in the same PR we avoid modifying the same line twice (e.g. one commit adding something, one commit removing that something).
Is it possible to move those changes in this commit?
basically you add INCLUDE_MAXIM_PATTERN and INCLUDE_MAXIM_CORRECTED in this commit and then you remove these in the last one...

Reply:
I'll see if I can move the related changes when rebasing in the newline at end of file and misaligned := changes